packages/account-v2/src/components/base/utils.ts

```diff
@@ -0,0 +1,17 @@
+import React, { isValidElement } from 'react';
+
+export const reactNodeToString = function (reactNode: React.ReactNode): string {
```

Comment:
I'm thinking what is the usecase we have to convert a react node to a string, Even in combobox we can rely on values only to render then it seems over complicated

Reply:
Need to check the behaviour of Wallets component. The whole base folder is planned to be removed once we have proper components